framework/Cargo.toml

```diff
@@ -112,7 +112,7 @@ opt-level = 3
 rpath = false
 lto = true
 overflow-checks = true
-opt-level = 3
+opt-level = 's'
```

Comment:
Oh, can we also change the opt-level in all crate profiles so that it's not a source of confusion ?

Reply:
Workspace overrides profile of packages, I'll remove package profiles to remove confusion and warning from cargo

Reply:
> Workspace overrides profile of packages, I'll remove package profiles to remove confusion and warning from cargo

During our last audit Oak recommended having the profile in multiple places to ensure that even if one is remove somehow, the profile still applies (explicitly because of the overflow checks that could be omitted if not set)